src/components/App.js

```diff
@@ -1,12 +1,26 @@
-import React from 'react';
-import '../styles/App.css';
+import React, { useState } from "react";
+import Search from "./Search";
+import SearchResults from "./SearchResults";
+
+import "../styles/App.css";
+
+const App = () => {
+  const [searchResults, setSearchResults] = useState([]);
 
-function App() {
   return (
     <div className="App">
-      <p>Hello World</p>
+      <img
+        className="nasa-logo"
+        src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
+        alt="nasalogo"
```

Comment:
I would write this in human spacing and casing as it will be shown to the user should the image not render for any reason.